bench/Main.hs

```diff
@@ -1,12 +1,12 @@
 module Main where
 
 import qualified Data.Mutable.Array as Array
-import Data.Mutable.HashMap (hmbench)
-import Gauge
+import qualified Data.Mutable.HashMap as HashMap
+import Test.Tasty.Bench (defaultMain)
 
 main :: IO ()
 main = do
   defaultMain
-    [ hmbench,
-      Array.benchmarks
+    [ Array.benchmarks,
+      HashMap.benchmarks
```

Comment:
Any reason for switching these two around?

Reply:
I added some consistency around the import naming scheme, and I reordered them to follow both alphabetical order and the import order.

Reply:
I don't see much value in alphabetical ordering of the benchmarks. But it doesn't hurt either, I suppose.